StructuralPatterns/Adapter/StringAdapter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Adapter
+{
+    public class StringAdapter : IStringAdapter
+    {
+        private string _inputString;
+
+        public StringAdapter()
+        {
+            _inputString = String.Empty;
+        }
+
+        public string InputString
+        {
+            get => _inputString;
+            set => _inputString = value;
+        }
+        public void CountLetters(params char[] letters)
+        {
+            int counter = 0;
+            Console.WriteLine("__________\n" + _inputString);
+            for (int i = 0; i < letters.Length; i++)
+            {
+                counter += _inputString.Count(t => t.Equals(letters[i]));
```

Comment:
Your method is supposed to return just a number. Do not need to display anything on console, it should be program.cs responcibility.

Reply:
By the way, foreach in this case would look better.